types/file-saver-es/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for file-saver-es 2.0
+// Project: https://github.com/eligrey/FileSaver.js/
```

Comment:
It's not the real path of the fork project.

Reply:
Do oyu know th real path?

Reply:
Nvm, found it.